bunsen-avro/src/main/java/com/cerner/bunsen/avro/tools/GenerateSchemas.java

```diff
@@ -55,6 +56,11 @@ public static int main(String[] args) {
         .skip(1)
         .collect(Collectors.toMap(Function.identity(), item -> Collections.emptyList()));
 
+    // Additional resources with contained resources for testing
+    resourceTypeUrls
+        .put("http://hl7.org/fhir/us/core/StructureDefinition/us-core-medicationrequest",
+            ImmutableList.of("http://hl7.org/fhir/StructureDefinition/Provenance"));
+
```

Comment:
Since this is part of the `tools` code (and not really itself testing code), let's remove this addition. The prior test should be sufficient.

Reply:
reverted 2a1ef489bf2a1df2cde5950bcb0ea42c2350bc1b